srcpkgs/github-cli/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'github-cli'
+pkgname=github-cli
+version=0.5.5
+revision=1
+wrksrc="cli-${version}"
+# create_wrksrc=yes
```

Comment:
...

Reply:
Sorry missed that. Will fix it asap. But why did xlint pass?